gen3-integration-tests/services/requestor.py

```diff
@@ -116,9 +115,14 @@ def get_request_id(self, user: str):
 
     def get_request_status(self, request_id: str, user: str = "main_account"):
         """Gets the request_status for the user's request_id in requestor"""
-        status_res = requests.get(
-            f"{self.BASE_URL}/{request_id}", headers=pytest.auth_headers[user]
-        )
+        auth = Gen3Auth(refresh_token=pytest.api_keys[user], endpoint=self.BASE_URL)
+        access_token = auth.get_access_token()
+        headers = {
+            "Accept": "application/json",
+            "Authorization": f"bearer {access_token}",
+            "Content-Type": "application/json",
+        }
+        status_res = requests.get(f"{self.BASE_URL}/{request_id}", headers=headers)
```

Comment:
can we use auth curl here?

Reply:
We cant use curl here, as curl doesn't accept headers as input
https://uc-cdis.github.io/gen3sdk-python/_build/html/_modules/gen3/auth.html#Gen3Auth.curl

Reply:
why do we need the headers? it is a simple get call, no?